### PR TITLE
refactor: Gap Chart - 0 population --> not enough information

### PR DESCRIPTION
**Pull request by eldu**
- For the gap chart, say "Not enough information" if the population is 0

Screen Shot 2022-04-20 at 5 15 02 PM

~Is the tooltip too awkward? Should there just be no tooltip?~
Tooltip removed -- previously [image](https://user-images.githubusercontent.com/5270855/164324261-7be20bc7-68e9-4f74-93d7-b63ac1d79e24.png)

### CODE REVIEW

File: src/components/dashboard/GapChart.vue [hunk 1]
```diff
@@ -140,7 +146,10 @@ const spec = computed(() => {
         encode: {
           enter: {
             x: { scale: "xscale", field: "pct" },
-            x2: { scale: "xscale", value: expected.value },
+            x2: {
+              scale: "xscale",
+              value: `datum.population > 0 ? ${expected.value} : 0`,
```
Comment (mcmcgrath13) [inline src/components/dashboard/GapChart.vue:151]: I think you'll want to add `strokeWidth` here and on the bar and make it conditional on the population as well - that should get rid of the faint amount of mark left

File: src/components/dashboard/GapChart.vue [hunk 2]
```diff
@@ -160,11 +160,11 @@ const spec = computed(() => {
           },
           update: {
             tooltip: {
-              signal: `{
+              signal: `datum.population > 0 ? {
```
Comment (mcmcgrath13) [inline src/components/dashboard/GapChart.vue:163]: does this work using the `{signal: '...', test: '...'}` style? that might be more consistent with the rest of the spec if it works

Comment (eldu) [reply src/components/dashboard/GapChart.vue]: Yes! I was able to use the test style!

File: src/components/dashboard/GapChart.vue (hunk 1 above)
Comment (eldu) [reply src/components/dashboard/GapChart.vue]: I changes this line back. And instead used a test on the stoke

**Review by mcmcgrath13 — APPROVED** [review]
🧪

**Pull request merged**